pages/[id]/delegate.tsx

```diff
@@ -64,62 +67,70 @@ const Delegate: NextPage = () => {
       <Head>
         <title>Cross Chain Voting: Delegate</title>
       </Head>
-      <div className="flex justify-center align-center self-center h-full w-1/3">
+      <div className="flex justify-center align-center self-center h-full w-1/2">
```

Comment:
```suggestion
      <div className="flex justify-center align-center self-center h-full max-w-lg">
```
I think it looks better if the bridge page box and the delegate page box are the same width.